CONTRIBUTING.rst

```diff
@@ -109,14 +108,14 @@ a new pull request against the `corresponding docs section of the MLflow code ba
 
 For more information about Plugins, see https://mlflow.org/docs/latest/plugins.html.
 
-Developing and testing changes to MLflow
-########################################
+Developing and testing MLflow
+#############################
 The majority of the MLflow codebase is developed in Python. This includes the CLI, Tracking Server,
 Artifact Repositories (e.g., S3 or Azure Blob Storage backends), and of course the Python fluent,
 tracking, and model APIs.
 
-Prerequisites
-~~~~~~~~~~~~~
+General prerequisites and dependencies
```

Comment:
Any reason to reword this? The original wording seemed a bit more concise

Reply:
I had taught about general (applies to all) and specific to individual components. For example, git applies to all; installing JavaScript dependencies is more specific to that component.

Reply:
Ah got it, sorry makes sense in the context of the surrounding content - maybe "Common prerequisites and dependencies"

Reply:
Yes, "Common prerequisites and dependencies" makes sense